scripts/docdiff

```diff
@@ -6,9 +6,11 @@ set -e
 if [ ! -f target/release/docdiff ]; then
   mkdir -p target/release
   cd common/docdiff
+
   cargo build --release
   cp target/release/docdiff ../../target/release/docdiff
   cd ../..
 fi
 
-./target/release/docdiff s2n-quic
+# disable unstable use warning since we dont want to docdiff unstable apis
+RUSTFLAGS="--cfg s2n_internal_dev" RUSTDOCFLAGS="--cfg s2n_internal_dev" ./target/release/docdiff s2n-quic
```

Comment:
testing locally.. we need the flags for docdiff. i think its because its being invoked from a 'external' crate, which is the behavior we want